src/invidious/comments.cr

```diff
@@ -328,7 +329,9 @@ def template_youtube_comments(comments, locale, thin_mode, is_replies = false)
       end
 
       author_name = HTML.escape(child["author"].as_s)
-
+      if child["verified"]?.try &.as_bool
+        author_name += " <i class=\"icon ion ion-md-checkmark-circle\"></i>"
```

Comment:
Not sure of the degree that Invidious wants to copy design elements from YT but I would like to note that YouTube would use 'checkmark-circle' for Verified comments by the uploader and would just use 'checkmark' for verified comments by other commentors

Reply:
@SamantazFox opinion? Do you want this in invidious?

Reply:
The more youtube-like features it has, the better ^^